src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,19 @@
-export const Tabs = () => {};
+export const Tabs = ({ tab, selectedTabId, onTabSelected, setTabItem }) => (
+  <li
+    data-cy="Tab"
+    className={`${selectedTabId === tab.id && 'is-active'}`}
+  >
+    <a
+      onClick={() => {
+        onTabSelected(tab.id);
+        setTabItem(tab);
```

Comment:
The `setTabItem` function seems to be not needed here. The `onTabSelected` function should be enough to handle the switching of tabs. This would make the Tabs component more stateless.